acceptance/lib/helper.rb

```diff
@@ -1 +1,3 @@
 $LOAD_PATH << File.expand_path(File.dirname(__FILE__))
+
```

Comment:
I understand what this is doing (I think :)), but there's no specific commentary as to why these changes are required. Does it make sense to add some (brief) info?

Reply:
It's more of a compatibility change than a something someone will maintain change, so I think the title covers it.